basic/README.md

```diff
@@ -1,59 +1,90 @@
 # templates-basic
 
 ## Summary
```

Comment:
Add some informations to summary section about scheduled job, why we have to use subscriptions.runForEachTenant regarding context.